Assets/Scenes/MAIN/MAIN.unity

```diff
@@ -476,7 +476,7 @@ Mesh:
   m_CorrespondingSourceObject: {fileID: 0}
```

Comment:
What did you add to the scene? Does it need to be included (mostly asking just to save conflicts with my PR once I merge it)?

Reply:
I just added a nav mesh agent to the NPC that gives the first mission because of an NPC now being able to follow a target.